next/public/locales/en/common.json

```diff
@@ -40,6 +41,10 @@
   "statusmessage-success": "Success",
   "statusmessage-warning": "Warning",
   "statusmessage-error": "Error",
+  "language-switcher-button": "Language switcher button",
```

Comment:
If it's a button, screenreaders will announce it as such, so we can remove the text "button" from the translation itself

Reply:
You were right here, made the change.